ncm-opennebula/src/main/pan/components/opennebula/schema.pan

```diff
@@ -521,6 +541,7 @@ type opennebula_untouchables = {
     "datastores" ? string[]
     "vnets" ? string[]
     "users" ? string[]
+    "groups" ? string[]
```

Comment:
I'm not sure about this change, there are only 5 resource types in OpenNebula so I have created a list just to keep it as simple as possible. The resource is gathered from the component with `$untouchables->{$type}`. I don't know how to change this by a dictionary and keep the backwards compatibility... You have to change your site schema, do you agree with that?